src/reducers/User.ts

```diff
@@ -11,14 +11,20 @@ const defaultState = {
   },
 }
 
-export const user = (state: any, action: any) => {
+export const user = (state: any = defaultState, action: any = {}) => {
   switch (action.type) {
     case Actions.SignIn.SIGN_IN_SUCCESS:
       return {
         ...state,
         ...action.payload,
         ...{ profile: { ...defaultState.profile, ...action.payload.profile } },
       }
+    case Actions.SignUp.SIGN_UP_SUCCESS:
+      return {
+        ...state,
+        ...action.payload,
```

Comment:
This looks like a great place for bugs to hide. I see this pattern was already in use before this PR, so I created [this ticket](https://github.com/poetapp/explorer-web/issues/181) which we can address separately, not to hold this PR that much.